specification/consumption/resource-manager/Microsoft.Consumption/stable/2021-05-01/examples/LotsListByBillingProfile.json

```diff
@@ -13,37 +13,69 @@
             "name": "lot1",
             "type": "Microsoft.Consumption/lots",
             "properties": {
+              "creditCurrency": "USD",
+              "billingCurrency": "USD",
               "originalAmount": {
                 "currency": "USD",
                 "value": 5000.00
               },
+              "originalAmountInBillingCurrency": {
+                "exchangeRate": 5000.00,
+                "exchangeRateMonth": 1
+              },
               "closedBalance": {
                 "currency": "USD",
                 "value": 60.90
               },
+              "closedBalanceInBillingCurrency": {
+                "exchangeRate": 5000.00,
+                "exchangeRateMonth": 1
+              },
+              "reseller": {
+                "resellerId": "/providers/Microsoft.Billing/billingAccounts/1234:5678/billingProfiles/2468/providers/Microsoft.Consumption/reseller/reseller1",
+                "resellerDescription": 1
+              },
               "source": "PurchasedCredit",
               "startDate": "2021-05-01T00:00:00Z",
               "expirationDate": "2021-05-01T00:00:00Z",
-              "poNumber": "3524"
+              "poNumber": "3524",
+              "purchaseDate": "2019-09-01T00:00:00Z",
+              "status": "Active"
             }
           },
           {
             "id": "/providers/Microsoft.Billing/billingAccounts/1234:5678/billingProfiles/2468/providers/Microsoft.Consumption/lots/lot2",
             "name": "lot2",
             "type": "Microsoft.Consumption/lots",
             "properties": {
+              "creditCurrency": "USD",
+              "billingCurrency": "USD",
               "originalAmount": {
                 "currency": "USD",
                 "value": 6000.00
               },
+              "originalAmountInBillingCurrency": {
+                "exchangeRate": 5000.00,
+                "exchangeRateMonth": 1
+              },
               "closedBalance": {
                 "currency": "USD",
                 "value": 80.90
               },
+              "closedBalanceInBillingCurrency": {
+                "exchangeRate": 5000.00,
+                "exchangeRateMonth": 1
+              },
+              "reseller": {
+                "resellerId": "/providers/Microsoft.Billing/billingAccounts/1234:5678/billingProfiles/2468/providers/Microsoft.Consumption/reseller/reseller2",
+                "resellerDescription": 2
+              },
               "source": "PurchasedCredit",
               "startDate": "2021-05-01T00:00:00Z",
               "expirationDate": "2019-12-31T00:00:00Z",
-              "poNumber": "31224"
+              "poNumber": "31224",
+              "purchaseDate": "2019-09-01T00:00:00Z",
```

Comment:
purchaseDate and status won't be available at the billingProfile scope.

Reply:
Removed.